tests/PHPStan/Rules/Preg/data/valid-regex-pattern.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/** @var string $string */
```

Comment:
This does not create a new variable so `$string` is resolved to ErrorType here. You can write an anonymous function with `strings $string` as a parameter.